extensions/wikidata/src/org/openrefine/wikidata/schema/entityvalues/ReconItemIdValue.java

```diff
@@ -33,6 +33,15 @@ public ReconItemIdValue(Recon recon, String cellValue) {
         super(recon, cellValue);
     }
 
+	@Override
+	public boolean isPlaceholder() {
+		if (isNew()) {
+            return true;
+		} else {
+            return false;
+		}
```

Comment:
Here you can just say `return isNew()`. Also, the same change should be done for the others: `ReconPropertyIdValue` and `ReconMediaInfoIdValue`.